aiogram/contrib/fsm_storage/sqlite.py

```diff
@@ -0,0 +1,93 @@
+import json
+import typing
+import sqlite3
+from ...dispatcher.storage import BaseStorage
+
+
+class SqliteStorage(BaseStorage):
+    def __init__(self, db_name='aiogram_fsm_storage.db', tbl_name='aiogram_fsm'):
+        self._db_name = db_name
+        self._tbl_name = tbl_name
+        self._conn = sqlite3.connect(db_name)
```

Comment:
Sync storage may affect performance and block event loop. Also, `sqlite3` is not really designed for apps like bots and, if I remember correctly, it has a write lock, which can also hit the performance.

Reply:
Maybe, but i think aiogram should have sqlite_storage. Not all bots need a high load.

Reply:
Every sqlite3 request will freeze ALL YOUR APPLICATION. It's not about high load.

Reply:
But SQLite is quite a fast thing. I don't see a problem in freezing the application for this amount of time. But it is useful for quick development. What's the need of installing Redis, for example, when you are the only supposed bot user? Personally, I have a couple of bots for personal use and I use my own implementation of SQLite for them, because I don't have too much time to manage Redis